napari_cellseg3d/code_models/model_framework.py

```diff
@@ -94,6 +96,17 @@ def __init__(
         available_devices = ["CPU"] + [
             f"GPU {i}" for i in range(torch.cuda.device_count())
         ]
+        from napari_cellseg3d.utils import _is_mps_available
+
+        try:
+            if _is_mps_available(torch):
+                from os import environ
+
+                environ["PYTORCH_ENABLE_MPS_FALLBACK"] = "1"
```

Comment:
It ought to be set globally as soon as MPS is available... Maybe if I move it; anyway this is not ideal, I don't want you to lost too much time in tests. Does server 6 have MPS ?

Reply:
it does, but let's focus on the other items first (unifying naming to wnet3d, etc :).